robosuite/models/objects/generated_objects.py

```diff
@@ -12,6 +12,7 @@ class PotWithHandlesObject(MujocoGeneratedObject):
 
     def __init__(
         self,
+        name,
```

Comment:
can we make `name` as an optional argument and automatically generate a name when the argument is not provided?

Reply:
I think it is best to explicitly require a name - otherwise we would need to generate names that do not conflict with existing names by using the current system time for example. But I think in most use cases, these objects are created within environments that should be keeping track of their objects regardless